account_financial_report/wizard/general_ledger_wizard_view.xml

```diff
@@ -91,10 +107,6 @@
                 res_model="general.ledger.report.wizard"
                 src_model="res.partner"
                 view_mode="form"
-                context="{
-                    'default_receivable_accounts_only':1,
-                    'default_payable_accounts_only':1,
-                    }"
```

Comment:
it is a pity to lost this context, what about add a hook to create two tags for account receivable and payable and set them as default